src/mono/wasm/debugger/DebuggerTestSuite/MiscTests.cs

```diff
@@ -932,20 +932,14 @@ public async Task InspectLocalsUsingClassFromLibraryUsingDebugTypeFull()
         [Theory]
         [InlineData(
             "DebuggerTests.CheckSpecialCharactersInPath", 
-            "debugger-test-special-char-in-path-%23%40/test%23.cs", 
             "dotnet://debugger-test-special-char-in-path.dll/test#.cs")]
         [InlineData(
             "DebuggerTests.CheckSNonAsciiCharactersInPath", 
-            "debugger-test-special-char-in-path-%23%40/non-ascii-test-ął.cs", 
             "dotnet://debugger-test-special-char-in-path.dll/non-ascii-test-ął.cs")]
         public async Task SetBreakpointInProjectWithSpecialCharactersInPath(
-            string classWithNamespace, string expectedLocationSufix, string expectedFileLocation)
+            string classWithNamespace, string expectedFileLocation)
         {
             var bp = await SetBreakpointInMethod("debugger-test-special-char-in-path.dll", classWithNamespace, "Evaluate", 1);
-            Assert.True(
-                bp.Value["breakpointId"].Value<string>().EndsWith(expectedLocationSufix), 
```

Comment:
What was the actual value that didn't match?

Reply:
Locally @# was coded into hex so %23%40 and on CI it was just @#.